packages/components/src/Actions/ActionSplitDropdown/ActionSplitDropdown.test.js

```diff
@@ -50,6 +50,34 @@ describe('ActionSplitDropdown', () => {
 		expect(wrapper).toMatchSnapshot();
 	});
 
+
+	it('should render items with icons', () => {
+		// given
+		const props = {
+			label: 'Add File',
+			icon: 'fa fa-plus',
+			onClick: jest.fn(),
+			items: [
+				{
+					label: 'From Local',
+					onClick: jest.fn(),
+					icon: 'fa fa-plus',
+				},
+				{
+					label: 'From Remote',
+					onClick: jest.fn(),
+					icon: 'fa fa-plus',
+				},
+			],
+		};
+
+		// when
+		const wrapper = renderer.create(<ActionSplitDropdown {...props} />).toJSON();
```

Comment:
please avoid as much as possible the use of renderer.create -> enzyme's shallow is better because it make your test really unit test. That is an integration tests, and they make some pains when low level component change (here Icon / action, ...)